organisms/organisation-unit-tree/new-features/select-relative-units.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Select units relative to the user's org unit, 'relative units'
+
+  Background:
+    Given the org unit tree has relative selection enabled
+
+  Scenario: Relative selection(s) is made
```

Comment:
This story lacks a `Then all relatively selected org units are displayed as selected` or sth similar.
I'm wondering whether org units selected this way should have a different selected state than non-relative selections (and a third selection style for both non-relative and relative selection)?

Reply:
Relative selections are made based on the users units, so it wouldn't make sense to select anything in the tree. Example:
```
Joe's Tree:
- Sierra Leone
---Bo
---Bombali
```
```
Jan-Gerke's Tree:
- Ghana
---Accra
---Kumasi
```

Relative selection `user sub-units` would result in 'Bo, Bombali' for Joe, 'Accra, Kumasi' for Jan-Gerke. I think making a selection in the tree would be misleading, those units are not actually being selected now, they may be selected at runtime.

This was my idea for dealing with this, simply disabling and making it clear that relative selections are a little bit different:
![image](https://user-images.githubusercontent.com/33054985/62113079-3d12a400-b2b4-11e9-96ad-eb3737eeef1a.png)

Reply:
Ok, sounds good to me